Source/Krypton Components/Krypton.Navigator/Navigator/KryptonPageActionList.cs

```diff
@@ -144,13 +144,13 @@ public string ToolTipBody
         /// </summary>
         public Bitmap? ImageSmall
         {
-            get => _page.ImageSmall;
+            get => _page!.ImageSmall;
 
             set
             {
-                if (_page.ImageSmall != value)
+                if (_page!.ImageSmall != value)
```

Comment:
if page is null and value is null, then line 154 will still throw it's toys out of the pram !

Reply:
I also noticed that , the code has been setup like that from the start and will fail in most areas if the cast from `owner.Component` to KControl should return null. Seems that this has always worked so far.

So `owner.Component` always has to be a valid reference for the action list code to run.
Putting in a check on the type cast of 'owner.Component' and throw will make use of the null-forgiving operator redundant. The control var can become non-nullable. Will process all Action Lists Like in the example below.
```cs
        private readonly KryptonPage _page;
        private readonly IComponentChangeService? _serviceComponentChange;
        private DesignerActionItemCollection _actions;
        #endregion

        #region Identity
        /// <summary>
        /// Initialize a new instance of the KryptonPageActionList class.
        /// </summary>
        /// <param name="owner">Designer that owns this action list instance.</param>
        public KryptonPageActionList(KryptonPageDesigner owner)
            : base(owner.Component)
        {
            // Remember designer and actual component instance being designed
            _page = owner.Component as KryptonPage ?? throw new NullReferenceException(GlobalStaticValues.ParameterCannotBeNull(nameof(owner.Component)));

            // Cache service used to notify when a property has changed
            _serviceComponentChange = GetService(typeof(IComponentChangeService)) as IComponentChangeService;
        }
        #endregion
        
        #region Public
        /// <summary>
        /// Gets and sets the page text.
        /// </summary>
        public string TextShort
        {
            get => _page.Text;

            set
            {
                if (!_page.Text.Equals(value))
                {
                    _serviceComponentChange?.OnComponentChanged(_page, null, _page.Text, value);
                    _page.Text = value;
                }
            }
        }

```